### PR TITLE
Add homebrew devel and HEAD notes to docs.

### PR DESCRIPTION
**Pull request by mikesplain**
Added to brew in https://github.com/Homebrew/homebrew-core/pull/20464

### CODE REVIEW

File: docs/update_kops.md [hunk 1]
```diff
@@ -2,12 +2,19 @@
 
 Update the latest source code from kubernetes/kops
 
-```
+```bash
```
Comment (chrislovecnm) [inline docs/update_kops.md]: Not on you, but I do not like this document much.  We have had issues with a lot of people installing from source in the past.  Wondering if we should refer them just include the base install instructions here that are in the README.

Comment (chrislovecnm) [reply docs/update_kops.md]: Or just add notes that we just replace the binary to update kops in the install doc, and link to that document in this doc.

File: docs/install.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ From Homebrew:
 brew update && brew install kops
 ```
 
+Development Releases and master can also be installed via Homebrew:
```
Comment (chrislovecnm) [inline docs/install.md]: Should we just add a link for developers to the compile and homebrew instructions?  I would like it to be another click for a developer to install, and not have people installing HEAD, unless they know they are now a developer.

Comment (mikesplain) [reply docs/install.md]: @chrislovecnm That's a good idea, I'll move this to a docs/development/install.md and link?

Comment (mikesplain) [reply docs/install.md]: Or I can add it in docs/development/building.md as an "easier" way to build and install?

File: docs/update_kops.md (hunk 1 above)
Comment (mikesplain) [reply docs/update_kops.md]: Okay that makes sense. I'll rework this pending the above comments ^.  We probably should just refer them to docs/development for the `git pull && make` version.

**Comment by mikesplain** [comment]
@chrislovecnm Let me know if this is more what you had in mind. Thanks!

**Comment by justinsb** [comment]
/lgtm

Thanks @mikesplain !

**Pull request merged**